bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -0,0 +1,547 @@
+                    setMode(config.url, loadpoint, command.toString());
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_MIN_SOC:
+                    setMinSoC(config.url, loadpoint, Integer.parseInt(command.toString().replaceAll(" %", "")));
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_TARGET_SOC:
+                    setTargetSoC(config.url, loadpoint, Integer.parseInt(command.toString().replaceAll(" %", "")));
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_TARGET_TIME:
+                    if (targetTimeEnabled == true) {
+                        targetTimeZDT = new DateTimeType(command.toString()).getZonedDateTime();
+                        setTargetCharge(config.url, loadpoint, targetSoC, targetTimeZDT);
+                        ChannelUID channel = new ChannelUID(getThing().getUID(), "loadpoint" + loadpoint,
+                                CHANNEL_LOADPOINT_TARGET_TIME);
+                        updateState(channel, new DateTimeType(targetTimeZDT));
+                    }
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_TARGET_TIME_ENABLED:
+                    if (command == OnOffType.ON) {
+                        targetTimeEnabled = true;
+                        setTargetCharge(config.url, loadpoint, targetSoC, targetTimeZDT);
+                    } else {
+                        targetTimeEnabled = false;
+                        unsetTargetCharge(config.url, loadpoint);
+                    }
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_PHASES:
+                    setPhases(config.url, loadpoint, Integer.parseInt(command.toString()));
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_MIN_CURRENT:
+                    setMinCurrent(config.url, loadpoint, Integer.parseInt(command.toString().replaceAll(" A", "")));
+                    refresh();
+                    break;
+                case CHANNEL_LOADPOINT_MAX_CURRENT:
+                    setMaxCurrent(config.url, loadpoint, Integer.parseInt(command.toString().replaceAll(" A", "")));
+                    refresh();
+                    break;
+            }
+        }
+
+        // Note: if communication with thing fails for some reason,
+        // indicate that by setting the status with detail information:
+        // updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+        // "Could not control device at IP address x.x.x.x");
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(EvccConfiguration.class);
+        updateStatus(ThingStatus.UNKNOWN);
+
+        if ("".equals(config.url)) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, "No host configured");
```

Comment:
Please provide this text in the i18n file **evcc.properties** so it can be localized.

Reply:
How would this exactly work? Could you please provide an example?

Reply:
Code:
```
updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, "@text/offline.configuration-error.no-host");
```

Properties:
```
offline.configuration-error.no-host = No host configured
```